viewer/packages/360-images/src/collection/Image360CollectionFactory.ts

```diff
@@ -28,35 +30,37 @@ export class Image360CollectionFactory<T> {
   public async create(
     dataProviderFilter: T,
     postTransform: THREE.Matrix4,
-    preMultipliedRotation: boolean
+    preMultipliedRotation: boolean,
+    reloadImage: (entity: Image360Entity, revision: Image360RevisionEntity) => Promise<void>
   ): Promise<DefaultImage360Collection> {
-    const event360Descriptors = await this._image360DataProvider.get360ImageDescriptors(
+    const historicalDescriptors = await this._image360DataProvider.get360ImageDescriptors(
       dataProviderFilter,
       preMultipliedRotation
     );
-    event360Descriptors.forEach(image360Descriptor => image360Descriptor.transform.premultiply(postTransform));
+    historicalDescriptors.forEach(image360Descriptor => image360Descriptor.transform.premultiply(postTransform));
 
-    const points = event360Descriptors.map(descriptor => new Vector3().setFromMatrixPosition(descriptor.transform));
+    const points = historicalDescriptors.map(descriptor => new Vector3().setFromMatrixPosition(descriptor.transform));
     const collectionIcons = new IconCollection(points, this._sceneHandler, this._onBeforeSceneRendered);
     const icons = collectionIcons.icons;
 
-    const entities = zip(event360Descriptors, icons)
+    const entities = zip(historicalDescriptors, icons)
       .filter(isDefined)
       .map(([descriptor, icon]) => {
         return new Image360Entity(
           descriptor,
           this._sceneHandler,
           this._image360DataProvider,
           descriptor.transform,
-          icon!
+          icon!,
```

Comment:
Don't think the `!` operator is nessesary anymore since it is type guarded by `isDefined`.